### PR TITLE
add redirect url for sendEmailVerification

### PR DESCRIPTION
**Pull request by jagaSto**
**What changes does this PR introduce**

* adds redirect URL for email verification, which allows the user to navigate back to the website
* does **not** automatically redirect the user to the website (this is not possible with the selected auth method, as far as my research suggested)

Screenshot 2020-04-21 at 23 11 04

### CODE REVIEW

**Comment by Maurice22** [comment]
Should we think about implementing our own frontend page that handles all the actions as explained here:
https://firebase.google.com/docs/auth/custom-email-handler

File: src/views/Signup.jsx [hunk 1]
```diff
@@ -40,7 +41,7 @@ export default () => {
 
     try {
       const signUpResult = await createUserWithEmailAndPassword(email, password);
-      await signUpResult.user.sendEmailVerification();
+      await signUpResult.user.sendEmailVerification({ url: `${baseUrl}/${returnUrl}` });
```
Comment (Maurice22) [inline src/views/Signup.jsx:44]: This is lacking the hash sign and hence does not redirect to the right location. Should be like in my suggestion.
```suggestion
      await signUpResult.user.sendEmailVerification({ url: `${baseUrl}/#/${returnUrl}` });
```

File: src/views/VerifyEmail.jsx [hunk 2]
```diff
@@ -19,7 +20,7 @@ export default () => {
 
   const resendEmail = async (e) => {
     e.preventDefault();
-    await user.sendEmailVerification();
+    await user.sendEmailVerification({ url: `${baseUrl}/ask-for-help` });
```
Comment (Maurice22) [inline src/views/VerifyEmail.jsx:23]: Same as above.
```suggestion
    await user.sendEmailVerification({ url: `${baseUrl}/#/ask-for-help` });
```

**Comment by jagaSto** [comment]
We can definitely think about it since I don't see a way to perform an automatic redirect with the current implementation! Nevertheless, I would still merge this, since it will probably take a bit until we have discussed and implemented the proposition :)

**Pull request merged**